### PR TITLE
fix: rename org name

### PR DESCRIPTION
**Pull request by fayasnoushad**
- Changed old name to new name
- Changed old year to new year

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Eddie Jaoude Community
+Copyright (c) 2021 EddieHub Community
```
Comment (suitupstan) [inline LICENSE:3]: I don't think the year has to be changed though.

**Review by gptkrsh — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (fayasnoushad) [reply LICENSE:3]: > 

But this is template

Comment (krishguptadev) [reply LICENSE:3]:
```suggestion
Copyright (c) 2020 EddieHub Community
```

Comment (suitupstan) [reply LICENSE:3]: > > 
> 
> But this is template

Ooops..

**Review by gptkrsh — APPROVED** [review]

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
```diff
- Copyright (c) 2021 EddieHub Community
+ Copyright (c) 2022 EddieHub Community
```
Please change the year from 2021 to 2022

**Pull request closed without merging**